app/config/security.yml

```diff
@@ -24,3 +24,4 @@ security:
         - { path: ^/register, role: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/resetting, role: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/admin/, role: ROLE_ADMIN }
+        - { path: ^/my-plants/, role: ROLE_USER }
```

Comment:
Excuse me, but it's not the case.

You wrote "my plants". That would be ok, but it's redundant somehow. It's obvious for user that plants are his, not someone else. Plant is a resource.

There is a nice convention you can start with. Let me show you an example:

```
/plants - all plants for current user (list of plants)
/plant/5 - plant with id 5
/plant/5/edit - edit plant with id 5
/plant/new - create new plant
```

You should keep your URIs clean and consistent. Trust me, you'll avoid trouble in the future (as code grows up).

Reply:
This approach is especially good when it comes to define secured pathes in `security.yml`. As you can see, there are just `^/plant/` and `^/plants` patterns that will do the work.

Reply:
You can even merge those patterns by replacing `/plants` with `/plant/all` or something similar.